examples/splitBoxplot.html

```diff
@@ -0,0 +1,52 @@
+<html>
+
+<head>
+	<title>Visualizations: split-boxplot</title>
+	<link href="example.css" rel="stylesheet">
+	<link href="chart.css" rel="stylesheet">
+</head>
+
+<body>
+	<h1>Split-Boxplot Example</h1>
+	<div>This plot shows a split boxplot</div>
+	<div style="width:30%" id="plot"></div>
+	<hr/>
+	<div>Data:</div>
+	<textarea id="chartData" style="width:600px; height:400px">
+	[
+		{
+			"target": {"min": 8, "LIF":12, "q1":19, "median":22, "q3": 29, "UIF":37, "max": 42},
+			"compare":{"min": 1, "LIF":5, "q1":12, "median":15, "q3": 22, "UIF":30, "max": 35}
+		}
+	]
+	</textarea>
+	<button id="reload">Reload</button>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.3/require.min.js"></script>
+	<script src="../build/config.js"></script>
+	<script>
+		requirejs(['atlascharts/split-boxplot'], function(splitBoxplot) {
+			var cartData;
+			var plot = new splitBoxplot();
+			var target = document.querySelector('#plot');
+
+			function refreshPlot() {
+				chartData = JSON.parse(document.querySelector("#chartData").value);
+				plot.render(chartData, target, 300,50, {
+					ticks: 4,
+					xLabel: "Type",
+					yLabel: "Count",
+					showLegend: true				
+				});
```

Comment:
Where are the legend and xLabel / yLabel shown?
![image](https://user-images.githubusercontent.com/10010071/45113656-eee33800-b118-11e8-9e95-b892dad8c3f9.png)

Reply:
For this implementation, I was not going to support X/Y labels. The code is there as a guide to where to implement, but I don't need those features at this time and didn't want to spend the time to implement it now.

Reply:
And what about a legend? If we don't have it, the UX for determining which box represents what data is not so good (it won't be possible w/o overlapping the boxes with mouse)

Reply:
so, i don't think we want to repeat a legend in each individual boxplot for the view you are going to use this for (you haev a stack of individual boxplots in each cell). So, I'd put a legend separately from these visulzations, and you can class them with the 'target' and 'compare' classes and style those elements appropirately.  Because they are css classes, everythign with the target and compre classes will be applied with the appropriate styles.